var/spack/repos/builtin.mock/packages/noversion-bundle/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class NoversionBundle(BundlePackage):
+    """Simple bundle package with no version and one dependency"""
```

Comment:
IMO it's worth mentioning in the docstring that this package should be rejected.

Reply:
Added comments for this and the other no-version example.